cypress/utils/filters.js

```diff
@@ -53,8 +53,18 @@ function applyFilters(filters, filtersConf) {
           .find('input[type=checkbox]')
           .check({ force: true });
       });
+      // close dropdown again
+      cy.get(FILTER_TOGGLE).click({ force: true });
```

Comment:
If I remove them from this function, it fails. I can explore which are the "required" ones and which not, but definitely some of them are still needed. Do not know why.

Reply:
Could you please report a bug? this is not ok because if we use force: true, then it means the elements are not actionable or visible... (can possibly be another code smell)

Reply:
Sure